shared/build.gradle.kts

```diff
@@ -25,7 +25,9 @@ kotlin {
             }
         }
         val androidMain by getting
+        val androidAndroidTestRelease by getting
```

Comment:
should this be `androidTestRelease?`

Reply:
The first Android is in reference to the KMM structure, the second because this is running the UI tests in AndroidTest. I think I’ll have to add androidTest for unit tests tho. Gets kinda confusing 😅